docs/guide/webpack.md

```diff
@@ -0,0 +1,127 @@
+# Webpack
+
+## Extending Webpack config
+
+If you need webpack to do things that Apostrophe don't by default, but you don't want to write your own config from scratch. You can simply extend the Apostrophe one that is used to transpile code from `ui/src` folder of your modules.
+For example if you want to add a new loader for a specific file type, or simply add some aliases for conveniences.
+
+It's possible to override the webpack config from any module. You can just add a `webpack` property at module root, like so:
+
+```javascript
```

Comment:
Check out the AposCodeBlock component for these, which includes a spot for the filename.